configure.ac

```diff
@@ -132,6 +132,12 @@ AC_CONFIG_FILES([Makefile
                  pyanaconda/Makefile
                  pyanaconda/version.py
                  pyanaconda/dbus/Makefile
+                 pyanaconda/modules/Makefile
+                 pyanaconda/modules/boss/Makefile
+                 pyanaconda/modules/foo/Makefile
+                 pyanaconda/modules/bar/Makefile
+                 pyanaconda/dbus_addons/Makefile
+                 pyanaconda/dbus_addons/baz/Makefile
```

Comment:
I think that @M4rtinK has the makefiles ready in his next pull request.

Reply:
Yep, I'll add it.